src/controllers/rolePresetsController.js

```diff
@@ -1,25 +1,29 @@
-const { hasPermission } = require('../utilities/permissions');
+const helper = require('../utilities/permissions');
 
 const rolePresetsController = function (Preset) {
   const getPresetsByRole = async function (req, res) {
-    if (!(await hasPermission(req.body.requestor, 'putRole'))) {
-      res.status(403).send('You are not authorized to make changes to roles.');
+    if (!(await helper.hasPermission(req.body.requestor, 'putRole'))) {
+      res.status(403).send({ error: 'You are not authorized to make changes to roles.' });
```

Comment:
Why have you changed the `send` argument from `'You are not authorized to make changes to roles.'`  into an object like this: `{ error: 'You are not authorized to make changes to roles.' }`. This could break the frontend application since it's expecting a string rather than an object to process an error?